lib/components/composed_component.dart

```diff
@@ -43,17 +43,4 @@ abstract class ComposedComponent extends Component {
     }
   }
 
-  void iterateOverComponents ( IterateOverComponents iterateOverComponentsCallback ){
-    components.forEach(iterateOverComponentsCallback);
-  }
-
-  @override
```

Comment:
I think you should implement Resizable and implement the children method to return this.components, in order to keep this behavior. Like so:
```dart
  List<Resizable> children() => this.components.where((r) => r is Resizable).cast<Resizable>();
```
or something like that (that's from the top of my head, I have not tested it!)